src/scss/sections/brands/brands.scss

```diff
@@ -0,0 +1,45 @@
+.brands {
+    margin: 0 -28px;
+    border: 1px solid #d8e3fe;
+
+    &-wrapp {
+        min-height: 144px;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        align-items: center;
+        padding-top: 20px;
+
+        @media all and (min-width: $md) {
+            padding-top: 0;
+        }
+    }
+
+    .brand {
+        display: flex;
+        justify-content: center;
+        width: calc( 100% / 2);
+        margin-bottom: 20px;
+        padding: 0 28px;
+
+        a:hover {
+			svg path {
+				fill: #6188ff;
+                opacity: 1;
+			}
+        }
```

Comment:
Please, check tabs spacing here. Looks weird, maybe it's GitHub bug.

Reply:
it's git hub bug